gempython/tools/amc_user_functions_uhal.py

```diff
@@ -172,3 +172,6 @@ def blockL1A(device):
 
 def enableL1A(device):
     writeRegister(device, "GEM_AMC.TTC.CTRL.L1A_ENABLE", 0x1)
+
+def getAMCL1Acount(device):
+    return readRegister(device, "GEM_AMC.TTC.CMD_COUNTERS.L1A")
```

Comment:
> I can make the functions for AMC and for OH with the same name and call them as module_name.function_name, but this might be ambigous

This is the "correct" way to do things, (the same as in `c` where you have to specify the namespace for conflicting function names.
You'd do something akin to:
```python
import <path to optohybrid_user_functions> as oh
import <path to amc_user_functions> as amc
...
amcTrigs = amc.getL1ACount(args)
ohTrigs = oh.getL1ACount(args)
```

The second option I reject outright